testing/trino-product-tests-launcher/src/test/java/io/trino/tests/product/launcher/util/TestConsoleTable.java

```diff
@@ -61,10 +61,10 @@ public void testRenderTableWithDifferentColumnWidths()
 
         assertThat(table.render()).isEqualTo(
                 "+-----------+-------+\n" +
-                "|     hello | world |\n" +
-                "|         a |     b |\n" +
-                "| pineapple | rocks |\n" +
-                "+-----------+-------+");
+                        "|     hello | world |\n" +
```

Comment:
Add a
```
"" +
```
line so that the expected table appears adjusted in the source

(same in other places in the file)